readinglist/resource.py

```diff
@@ -273,8 +285,112 @@ def _extract_sorting(self, queryparams):
                 direction = -1 if order == '-' else 1
                 sorting.append((field, direction))
 
+        if not modified_field_used:
+            # Add a sort by the ``modified_field`` in descending order
+            # useful for pagination
+            sorting.append((self.modified_field, -1))
```

Comment:
Well this is after ``-stored_on`` which is not unique. We will handle default sortering in #84 this won't impact :)